src/actions/guides/database/querying.cr

```diff
@@ -7,294 +7,240 @@ class Guides::Database::Querying < GuideAction
+    user = UserQuery.last
+    ```
 
-    For example:
+    Selecting the user with `id = 3` `SELECT COLUMNS FROM users WHERE id = 3 LIMIT 1`
 
     ```crystal
-    # The query is not yet run
-    query = UserQuery.new
-    query.name("Sally")
-    query.age(30)
-
-    # The query will run once `each` is called
-    # Results are not cached so a request will be made every time you call `each`
-    query.each do |user|
-      pp user.name
-    end
+    user = UserQuery.find(3)
     ```
 
-    ## Finding by ID
+    ### Select distinct / distinct on
 
-    ```crystal
-    id = 1
-    UserQuery.new.find(id)
+    `SELECT DISTINCT COLUMNS FROM users`
 
-    # You can also chain methods before calling find, first, etc.
-    UserQuery.new.age(30).find(1)
+    ```crystal
+    UserQuery.new.distinct
     ```
 
-    ## Ensuring no results are returned
+    Select distinct rows based on the `name` column `SELECT DISTINCT ON (users.name) FROM users`
 
     ```crystal
-    UserQuery.new.none
+    UserQuery.new.distinct_on(&.name)
     ```
 
-    This can be helpful when under certain conditions you want the resultd to be empty.
+    > Notice these two methods are called in the instance of `UserQuery`, and not the class.
+    > Aside from the quick select helper methods, the rest are instance methods.
 
-    > This method does not return an empty array immediately. You can still chain other query methods, but it
-      will always return no records. For example: `UserQuery.new.none.first` will never return a result
+    ## Where Queries
 
-    ## Shorthand methods
+    The `WHERE` clauses are the most common used in SQL. Each of the columns generated by the model
+    will give you a method for running a `WHERE` on that column. (e.g. the `age` column gives us `WHERE age = value` with the `age(value)` method.
 
-    There are a few methods that are used so often that we've added shortcut methods:
+    ### A = B
 
-    * `find`
-    * `first`
-    * `last`
+    `SELECT COLUMNS FROM users WHERE age = 54`
 
     ```crystal
-    id = 1
-    UserQuery.find(id)
-    UserQuery.first
-    UserQuery.last
+    UserQuery.new.age(54)
     ```
 
-    ## Type specific query methods
+    ### A = B AND C = D
 
-    The previous section mostly focused on simple equality queries, but you can also
-    do more advanced queries depending on the column type.
+    `SELECT COLUMNS FROM users WHERE age = 43 AND admin = true`
 
-    ### All types
+    ```crystal
+    UserQuery.new.age(43).admin(true)
+    ```
 
-    * `is`
-    * `desc_order`
-    * `asc_order`
+    > All query methods are chainable!
 
-    ### String
+    ### A != B
 
-    * `like`
-    * `ilike`
-    * `lower`
+    `SELECT COLUMNS FROM users WHERE name != 'Billy'`
 
-    ### Int32 and Time
+    ```crystal
+    UserQuery.new.name.not.eq("Billy")
+    ```
+
+    > The `not` method can be used to negate other methods like `eq`, `gt`, `lt`, `in`.
 
-    * `gt`
-    * `gte`
-    * `lt`
-    * `lte`
+    ### A gt/lt B
```

Comment:
It may be good to add a short sentence between the SQL and the heading that explains this in human readable terms so it is searchable. Something like `Where column A is greater than or less than column B`. That kind of thing